tools/fuchsia/debug_symbols.gni

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# The inputs to this template are 'binary_path' and a boolean 'unstripped'.
+# If 'unstripped' is specified, we append '.debug' to the symbols name.
+template("_copy_debug_symbols") {
+  assert(defined(invoker.binary_path), "'binary_path' needs to be defined.")
+
+  bin_path = invoker.binary_path
+
+  _args = []
+  if (defined(invoker.unstripped) && invoker.unstripped) {
+    _args += [ "--unstripped" ]
+  }
+
+  action(target_name) {
+    testonly = defined(invoker.testonly) && invoker.testonly
+
+    deps = []
+    if (defined(invoker.deps)) {
+      deps = invoker.deps
+    }
+
+    script = "$flutter_root/tools/fuchsia/copy_debug_symbols.py"
+
+    sources = [
+      bin_path,
+    ]
+
+    _dest_base =
+        "${root_out_dir}/flutter-debug-symbols-${target_os}-${target_cpu}"
+
+    args = _args + [
+             "--executable-name",
+             target_name,
+             "--executable-path",
+             rebase_path(bin_path),
+             "--destination-base",
+             rebase_path(_dest_base),
+           ]
+
+    outputs = [
+      "${_dest_base}/.${target_name}_success",
```

Comment:
I think the convention is `.stamp`. Also, why is the file hidden?

Reply:
I was making these hidden to avoid the noise when you try to inspect the directories. `.stamp` seems to have a special meaning as far as `gn` is concerned - there is a tool (" "stamp": Tool for creating stamp files") that gn uses to generate these, so I tried to avoid that. If that is the convention, i'm happy to change it to `.stamp`.